src/command_modules/azure-cli-network/azure/cli/command_modules/network/_validators.py

```diff
@@ -263,6 +267,9 @@ def process_ag_url_path_map_create_namespace(namespace): # pylint: disable=unuse
         namespace.default_http_settings = _generate_ag_subproperty_id(
             namespace, 'backendHttpSettingsCollection', namespace.default_http_settings)
 
+        # extract the certificate data from the provided file
+        namespace.cert_data = read_base_64_file(namespace.cert_data)
+
```

Comment:
Why is this here? App-gateway url-path-map create does not have a cert_data field. This should crash.